### ISSUE TITLE
no-jQuery is replacing all of parent element's HTML

### ISSUE THREAD

**Issue by azentz** [nikolalsvk/render_async#39]
In _render_async.html.erb for the no-jQuery condition, all of the HTML for a container's parent is being replaced.  I'm assuming it should only be replacing the container itself.

The line in question is: `container.parentNode.innerHTML = request.response;`

**Comment by walterdavis** [comment]
No, this is actually how you update the contents of an element without jQuery. If the outer parent was being replaced, the line would read `container.parentNode.outerHTML = request.response;`

**Comment by azentz** [comment]
I may not have explained what I'm seeing very well.  Yes, only the innerHTML of the outer parent is being replaced.  But that may include more elements than just the container.  For example, if you have:

```
<div id="parent">
    Text or elements here
    <%= render_async comment_stats_path %>
</div>
```

That would result in all of the HTML within the "parent" div to be replaced, including `Text or elements here`.  I was expecting only the container created by render_async to be replaced.

**Comment by nikolalsvk** [comment]
@azentz really sorry about this 😕 

I'll fix it ASAP and release the new version 🙇

**Comment by nikolalsvk** [comment]
@azentz released 1.1.3 version that should address this issue https://rubygems.org/gems/render_async

Can you please check it out if it works for you? 🍺

**Comment by azentz** [comment]
Works well for me.  Thank you very much.
